### PR TITLE
Add initial support for Markdown

### LINKED ISSUE #5: Add Markdown support
Modify the file handling so that it supports both .txt files as well as .md

### PR DESCRIPTION
**Pull request by Reza9472**
Linked issue: Closes #5  

As the program is already able to receive markdown files and create HTMLs from them, I have created a parseMarkdown() function that receives a string and adds the markdown features. 

Because the program is supposed to receive text files as well, I thought the best way to do this with minimum changes in the code, is to use regular expressions to parse the markdown features and add it to the header and body in the main.js.
This function parses: 

1. headers
2. blockquotes
3. bold texts
4. italics
5. images
6. links
7. line breaks

This function will not change the formatting of HTML files created from the text files.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -36,6 +36,22 @@ function checkExists(args, recursive) {
   }
 }
 
+// Markdown Support Function
```
Comment (AmasiaNalbandian) [inline src/main.js:39]: This is a great method! Works perfectly well! :)

File: src/main.js [hunk 2]
```diff
@@ -59,9 +75,9 @@ async function writeHTML(data, filename) {
     content.forEach((line) => {
       if(linecount ===0) {
         title = line;
-        dataForBody += `<h1>${line}</h1>\n`;
+        dataForBody += `<h1>${parseMarkdown(line)}</h1>\n`;
```
Comment (AmasiaNalbandian) [inline src/main.js:78]: In the code there is a function on line 113 called readFile, which is called from line 32 and passed the name from the file object. What I would do is I would check in the checkExists what the file type is, and then only if the type is md to parseMarkdown. As of now, ALL input files which exist are going through the parseMarkdown and this takes up resources, especially when none of the files are markdown.

Comment (AmasiaNalbandian) [reply src/main.js:78]: The object that gets passed into the createHTML function is of: 
{ 
name: 
type: 
}

**Review by AmasiaNalbandian — CHANGES_REQUESTED** [review]

**Comment by humphd** [comment]
@Reza9472 it looks like you have some more follow-up to do here, based on @AmasiaNalbandian's review.  Can you get that in soon please?  Also, `Closes Issue #9` should be `Closes #9`, see https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue.

File: src/main.js (hunk 2 above)
Comment (Reza9472) [reply src/main.js]: Thanks for the review.
I have updated my request.
Please check it out

Comment (AmasiaNalbandian) [reply src/main.js]: LGTM! Just needed to do one more function for the page title (in the tab). Thank you!

**Review by AmasiaNalbandian — APPROVED** [review]

**Pull request merged**